source/software/r_package_management.rst

```diff
@@ -133,7 +118,7 @@ Here, science is the name of the environment you want to work in.
 Install an additional package
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To install an additional package, e.g., \`pandas`, first ensure that the
+To install an additional package, e.g., \`rodbc`, first ensure that the
```

Comment:
instead of \`rodbc`, you need to put `rodbc` in between opening/closing double-backticks.